run_analysis.py

```diff
@@ -37,6 +41,120 @@ def update_generate(config, generate):  # {{{
     config.set('output', 'generate', generateString)  # }}}
 
 
+def run_parallel_tasks(config, analyses, configFiles, taskCount):
```

Comment:
I am not reviewing these as I am not sure what I could add: I just assume that's what you need to do for running parallel tasks.